qemu/tests/blockdev_stream_cor_base.py

```diff
@@ -0,0 +1,106 @@
+import json
+import time
+
+from virttest import data_dir
+
+from provider.blockdev_stream_base import BlockDevStreamTest
+from provider.virt_storage.storage_admin import sp_admin
+from provider import backup_utils
+
+
+class BlockdevStreamCORBase(BlockDevStreamTest):
+    """Do block-stream with copy-on-read filter as base """
+
+    def __init__(self, test, params, env):
+        super(BlockdevStreamCORBase, self).__init__(test, params, env)
+        self.snapshot_chain = params["snapshot_chain"].split()
+
+    def prepare_snapshot_file(self):
+        self.trash = []
```

Comment:
add snapshot images into trash, but never remove them

Reply:
Have updated, remove image in post_test. Thanks zhenchao.